src/trigger/climbable.cpp

```diff
@@ -136,9 +133,17 @@ Climbable::event(Player& player, EventType type)
       }
     }
   }
+
   if (type == EVENT_LOSETOUCH) {
     player.stop_climbing(*this);
-    climbed_by = nullptr;
+    auto it = climbed_by.begin();
```

Comment:
We don't use c++20 yet, do we? Otherwise, you could've used std::erase_if

Reply:
We don't yet, indeed